test/test_site.rb

```diff
@@ -175,7 +175,6 @@ def generate(site)
         method.call(*args, &block).reverse
       end
       @site.process
-      # files in symlinked directories may appear twice
```

Comment:
Maybe say here than symlinked directories are excluded from this list and added in when not Windows.